src/db/XMediaSchema.ts

```diff
@@ -0,0 +1,47 @@
+import mongoose from 'mongoose'
+import mongooseLeanVirtuals from 'mongoose-lean-virtuals'
+import muuid from 'uuid-mongodb'
+
+import { XMediaType } from './XMediaTypes.js'
+
+const { Schema } = mongoose
+
+const XMediaSchema = new Schema<XMediaType>({
+  userId: {
```

Comment:
User id in Auth0 is in uuid v4 format (Ex: `96373086-2d70-4e92-8036-3947d4cec3f9`) so we'd need to use MUUID here.  See `mediaUuid`  below for declaration.

Reply:
I've made this change and also changed destinationId on XMedia Schema to also use MUUID since this is what Media Schema is using. Lastly, I added onModel to XMedia (also copied from Media Schema).